twinTrim/utils.py

```diff
@@ -8,7 +8,7 @@
 def handle_and_remove(filepath):
     try:
         os.remove(filepath)
-        click.echo(click.style(f"Deleted: {filepath}", fg='green'))
+        click.echo(click.style(f"Deleted: {filepath}"))
```

Comment:
do not change the utils file unless needed. Your task issue mentioned falgs.py only.